Example/ios/Example/AppDelegate.m

```diff
@@ -31,7 +31,7 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
    * on the same Wi-Fi network.
    */
 
-  jsCodeLocation = [NSURL URLWithString:@"http://192.168.179.57:8081/index.ios.bundle?platform=ios&dev=true"];
+  jsCodeLocation = [NSURL URLWithString:@"http://192.168.50.35:8081/index.ios.bundle?platform=ios&dev=true"];
```

Comment:
Nitpick: Could we leave the IP at its original state please? 😄 
At some point I need to update the example to React Native 0.29+ anyway which gets rid of defining the defining the IP in `AppDelegate` anyway.